src/feedback.rs

```diff
@@ -20,6 +20,32 @@ static FLATTENED_FILENAME_REGEX: LazyLock<Regex> = LazyLock::new(|| {
     Regex::new("@student\\.kit\\.edu_\\d+-(?<filename>.*)").expect("Invalid regex pattern")
 });
 
+pub fn upload_points(grade_page: &GradePage, ilias_client: &IliasClient) -> Result<(), Whatever> {
+    let mut changed_submissions = vec![];
+    while let Some(selected_student) = FuzzySelect::with_theme(&ColorfulTheme::default())
+        .with_prompt("Pick a student")
+        .default(0)
+        .items(grade_page.submissions.as_slice())
+        .interact_opt()
+        .unwrap()
+    {
+        let selected_points: String = dialoguer::Input::with_theme(&ColorfulTheme::default())
+            .with_prompt("Punkte:")
+            .interact_text()
+            .expect("Interaction failed");
```

Comment:
This should also be `.whatever_context(...)?;`. Interactions in feedback/download stem from a time before sensible error handling, but we have that now \o/